lab_8_llm/main.py

```diff
@@ -34,6 +30,7 @@ class RawDataImporter(AbstractRawDataImporter):
     """
     A class that imports the HuggingFace dataset.
     """
+    _raw_data: DataFrame
```

Comment:
Delete and add in code a check: _raw_data containt None? If yes, then rise error

Reply:
Done (implemented a check in start.py)